### ISSUE TITLE
`deno fmt` modifies ast with sparse arrays

### ISSUE THREAD

**Issue by 102** [denoland/deno#17483]
`deno fmt` removes the trailing empty slots in sparse arrays, when it is expected that this will only change formatting, e.g. given the following input:

```typescript
[, ,];
[1, ,];
[1, , 3];
```
`deno fmt` without any settings will return the following output, updating the first and second lines in an unexpected way:
```typescript
[];
[1];
[1, , 3];
```
---------
deno version information:
```
% deno --version

deno 1.23.0 (release, aarch64-apple-darwin)
v8 10.4.132.5
typescript 4.7.2
```

**Comment by 102** [comment]
It seems that this is a part of dprint_plugin_formatting strategy, though I'm not sure why is it required https://github.com/dprint/dprint-plugin-typescript/blob/697da587116633b939057112ff9e3f34a12afa29/tests/specs/patterns/ArrayPattern/ArrayPattern_All.txt#L18-L30

**Comment by 0f-0b** [comment]
The fix is included in Deno since [1.26.0](https://github.com/denoland/deno/releases/tag/v1.26.0).

**Comment by littledivy** [comment]
Fixed in recent Deno version. Upgrade using `deno upgrade`
